syft/frameworks/crypten/context.py

```diff
@@ -79,12 +82,30 @@ def _send_party_info(worker, rank, msg, return_values):
     return_values[rank] = response.contents
 
 
-def toy_func():
-    # Toy function to be called by each party
-    alice_t = crypten.cryptensor([73, 81], src=0)
-    bob_t = crypten.cryptensor([90, 100], src=1)
-    out = bob_t.get_plain_text()
-    return out.tolist()  # issues with putting torch tensors into queues
+def toy_func(*args, **kwargs):
+    # !! This might work only because we have virtual workers
```

Comment:
We shouldn't perform workers operations on crypten functions I think. We should try at best to make those operations transparent by handling them via function decorators or our own crypten.load function

Reply:
I will look at the ```crypten.load``` how it works and try something from there